src/commands/monitor.rs

```diff
@@ -69,7 +70,16 @@ pub async fn run(opts: Monitor) -> Result<()> {
                     class.push(status.into());
                     let text = match (track.artist, track.title) {
                         (Some(artist), Some(title)) => {
-                            format!("{} {} {}", artist, separator, title)
+                            let status_icon = match status {
+                                TrackStatus::Playing => " ",
+                                TrackStatus::Paused => " ",
+                                _ => "" // unable to get player state
+                            };
+                            output_format
+                                .replace("{artist}", &artist)
+                                .replace("{title}", &title)
+                                .replace("{separator}", separator)
+                                .replace("{status}", status_icon)
```

Comment:
Could you move this to `OutputFormatter`? If the replacing is done on all `Output.text`, we could make the other messages below configurable as well.

Additionally, the tooltip can be configurable.

You'll need to add a `template_fields: HashMap<String, String>` or something similar where you pass what needs replacing. Otherwise the `OutputFormatter` won't know.